scripts/plot_snd_timing.py

```diff
@@ -0,0 +1,68 @@
+"""
+Script designed to collect and output network traffic statistics.
+"""
+import pathlib
+
+import click
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import tcpdump_processing.convert as convert
+import tcpdump_processing.extract_packets as extract_packets
+
+
+class SRTDataIndex:
+	def __init__(self, srt_packets):
+		self.ctrl_pkts        = (srt_packets['srt.iscontrol'] == 1)
+		self.data_pkts = (srt_packets['srt.iscontrol'] == 0)
+		self.data_pkts_org = self.data_pkts & (srt_packets['srt.msg.rexmit'] == 0)
+
+
+@click.command()
+@click.argument(
+	'path', 
+	type=click.Path(exists=True)
+)
+@click.option(
+	'--overwrite/--no-overwrite',
+	default=False,
+	help=	'If exists, overwrite the .csv file produced out of the .pcap (or .pcapng) '
+			'tcpdump trace one at the previous iterations of running the script.',
+	show_default=True
+)
+def main(path, overwrite):
+	"""
+	This script parses .pcap or .pcapng tcpdump trace file captured at the receiver side, 
```

Comment:
Misleading description. Sounds like you haven't changed it and simply copied from get_traffic_stats.